mdp/hinet/switchboard.py

```diff
@@ -486,21 +479,16 @@ def __init__(self, in_channels_xy, field_channels_xy,
                 for y_in_chan in range(y_start_chan,
                                        y_start_chan + self.field_channels_xy[1]):
                     for x_in_chan in range(x_start_chan,
-                                       x_start_chan + self.field_channels_xy[0]):
+                                           x_start_chan + self.field_channels_xy[0]):
                         first_in_con = (in_trans.image_to_index(
-                                                    x_in_chan, y_in_chan) *
+                            x_in_chan, y_in_chan) *
                                         in_channel_dim)
-                        connections[first_out_con:
-                                    first_out_con + in_channel_dim] = \
-                            list(range(first_in_con,
-                                  first_in_con + in_channel_dim))
+                        connections[first_out_con:first_out_con + in_channel_dim] = list(
+                            range(first_in_con, first_in_con + in_channel_dim))
                         first_out_con += in_channel_dim
-        super(DoubleRect2dSwitchboard, self).__init__(
-                                input_dim=in_channel_dim *
-                                    in_channels_xy[0] * in_channels_xy[1],
-                                connections=connections,
-                                out_channel_dim=out_channel_dim,
-                                in_channel_dim=in_channel_dim)
+        super(DoubleRect2dSwitchboard, self).__init__(input_dim=in_channel_dim * in_channels_xy[0] * in_channels_xy[1],
```

Comment:
You are right about the 120 value! Strange that they don't use pep8 value as the default. Thanks for pointing this out. I would vote for pep8 limit of 79, since we are anyway following pep8 convention for the rest.